htdocs/product/inventory/card.php

```diff
@@ -180,7 +183,7 @@
 	if ($backtopage) {
 		print '<input type="hidden" name="backtopage" value="'.$backtopage.'">';
 	}
-	if ($backtopageforcancel) {
+	if (isset($backtopageforcancel)) {
```

Comment:
In top of page we may have
$backtopageforcancel = GETPOST('backtopageforcancel')
In such a case, the var is defined, empty bu defined.
A test if (!empty())  seems better.

Reply:
changed to !empty